app/main/routes_zymatic_api.py

```diff
@@ -53,6 +53,10 @@ def process_first_setup(args):
 @main.route('/API/zymaticFirmwareCheck')
 @use_args(zymatic_firmware_check_args, location='querystring')
 def process_zymatic_firmware_check(args):
```

Comment:
not positive this is the first or one of the first machine initialization requests, if someone with a Zymatic could double check with a PCAP or MITM log that would be awesome... though quite certain either this or `/API/usersetup` (effectively 'is machine registered') are in that.